.github/workflows/playwright-changed-files.yml

```diff
@@ -74,10 +74,15 @@ jobs:
         run: |
           cd e2e_playwright;
           rm -rf ./test-results;
-          pytest ${{ steps.changed-files.outputs.all_changed_files }} --browser webkit --browser chromium --browser firefox --video retain-on-failure --screenshot only-on-failure --full-page-screenshot --tracing retain-on-failure --output ./test-results/ -n auto --durations=5 -r aR -v
+          pytest ${{ steps.changed-files.outputs.all_changed_files }} --browser webkit --browser chromium --browser firefox --video retain-on-failure --screenshot only-on-failure --full-page-screenshot --tracing retain-on-failure --output ./test-results/ --cov=streamlit --cov-report=html -n auto --durations=5 -r aR -v
       - name: Upload failed test results
         uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright_test_results
           path: e2e_playwright/test-results
+      - name: Upload coverage report
+        uses: actions/upload-artifact@v4
```

Comment:
According to the docs, the retention period by default is 90 days (see [here](retention-days: 5)). For these reports perhaps we should reduce the retention period, maybe to a week or so to prevent too much data & cost. What do you think?

Reply:
sounds good, I updated it to a week.